ODPi-Runtime.md

```diff
@@ -234,6 +262,8 @@ custom-to-the-application configuration file, etc) that does not impact the ODPi
 
 -   Applications SHOULD either launch via the Apache Hadoop YARN ResourceManager REST API or via `${HADOOP_YARN_HOME}/bin/yarn jar`
 
+-   Applications SHOULD use JDBC, ODBC, or SQL to determine the structure of metadata rather than directly querying the metastore thrift interface whenever possible.
```

Comment:
Is it understood enough that this would be via Hive, or should that be called out?